client/app/account/billing/autoRenew/disableAutoRenewDomains/billing-autoRenew-disableAutoRenewDomains.controller.js

```diff
@@ -0,0 +1,32 @@
+angular
+    .module("Billing.controllers")
+    .controller("billingAutoRenewDisable", class BillingAutoRenewDisable {
+        constructor ($scope, $translate, BillingAutoRenew, Alerter, AUTORENEW_EVENT) {
```

Comment:
Params could be set in alphabetical order

Reply:
Indeed